macro/EXP1904_H7/sim/H7.C

```diff
@@ -0,0 +1,42 @@
+void H7() { 
```

Comment:
In my opinion, It is convenient to see an output characteristic name in a macro name.